packages/core/src/components/Dropdown/__stories__/dropdown.stories.js

```diff
@@ -299,9 +302,7 @@ export const AsyncOptions = {
         <Dropdown asyncOptions={fetchUserOptions} placeholder="Async options" cacheOptions defaultOptions />
       </div>
     );
-  },
-
-  name: "Async Dropdown"
```

Comment:
I've removed it intentionally and changed the name of the exported story to AsyncDropdown instead of AsyncOptions, so the name of the story now should be the same as the old one